docs/guides/extracting_intermediates.rst

```diff
@@ -267,7 +273,7 @@ your model more explicitly.
 Extracting gradients of intermediate values
 ===========================================
 For debugging purposes, it can be useful to extract the gradients of intermediate values.
-This can be done by using the ``.perturb()`` method over the desired values.
+This can be done by using the :meth:`Module.perturb() <flax.linen.Module.perturb>` method over the desired values.
```

Comment:
This is an auto-link to the documented method, if I understand correctly?

Reply:
Yup! It can be viewed [here](https://flax--2685.org.readthedocs.build/en/2685/guides/extracting_intermediates.html#extracting-gradients-of-intermediate-values).